packages/cli/src/lib/dataset.ts

```diff
@@ -38,20 +47,20 @@ const dataset = async ({ type }) => {
 
     let imageNum = 1;
     let annotationNum = 1;
-    const categoryMap = {};
+    const categoryMap: CategoryMap = {};
     let categoryNum = 1;
     for (let i = 0; i < imageDirs.length; i++) {
       const imageDir = imageDirs[i];
       try {
         fse.lstatSync(imageDir).isDirectory();
-        let dirName = imageDir.split(path.sep);
+        let dirName: string[] | string = imageDir.split(path.sep);
```

Comment:
because line 57:
```js
 fse.lstatSync(imageDir).isDirectory();
 let dirName: string[] | string = imageDir.split(path.sep);
 dirName = dirName[dirName.length - 1];
```
dirName is assigned as a string :(

Reply:
resolved:)